.github/actions/build-collection/action.yaml

```
@@ -0,0 +1,26 @@
+# Copyright 2024 Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: build-collection
+description: Builds the collection
+
+runs:
+  using: composite
+  steps:
+    - name: Download certificate
+      shell: bash
+      run: mkdir $FILES_DIR && wget https://ca.dynatrace.com/dt-root.cert.pem -P $FILES_DIR
+    - name: Building the collection
+      shell: bash
+      run: ansible-galaxy collection build .
```

```suggestion
      run: ansible-galaxy collection build . -vvv
```
`ansible-galaxy collection build .` does not generally provide much information.